flixel/addons/effects/FlxEffectFade.hx

```diff
@@ -0,0 +1,154 @@
+package flixel.addons.effects;
+
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import openfl.display.BitmapData;
+import openfl.geom.Point;
+
+/**
+ * This gradually fills the bitmapData with a color.
+ * 
+ * @author adrianulima
+ */
+class FlxEffectFade implements IFlxEffect
```

Comment:
I basically used the same approach as `FlxCamera#fade`.

and also `FlxEffectFade` can be in the middle of `FlxEffectSprite#effects` chain, this may have some use cases.